csharp/src/Apache.Arrow/Apache.Arrow.csproj

```diff
@@ -2,27 +2,46 @@
 
   <Import Project="../../build/Common.props" />
 
+  <!-- Compile properties -->
   <PropertyGroup>
     <TargetFrameworks>netstandard1.3;netcoreapp2.1</TargetFrameworks>
     <LangVersion>7.2</LangVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <Authors>Apache</Authors>
+    <DefineConstants>$(DefineConstants);UNSAFE_BYTEBUFFER;BYTEBUFFER_NO_BOUNDS_CHECK;ENABLE_SPAN_T</DefineConstants>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\ApacheArrow.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <!-- AssemblyInfo properties -->
+  <PropertyGroup>
     <Product>Apache Arrow library</Product>
     <Copyright>2018 Apache Software Foundation</Copyright>
-    <PackageProjectUrl>https://fzcorp.visualstudio.com/digital-products</PackageProjectUrl>
-    <RepositoryUrl>https://fzcorp.visualstudio.com/digital-products/_git/fz-arrow</RepositoryUrl>
-    <RepositoryType>git</RepositoryType>
-    <PackageTags>apache arrow</PackageTags>
     <Company>Apache</Company>
-    <Version>0.0.1</Version>
-    <DefineConstants>$(DefineConstants);UNSAFE_BYTEBUFFER;BYTEBUFFER_NO_BOUNDS_CHECK;ENABLE_SPAN_T</DefineConstants>
+    <VersionPrefix>0.13.0</VersionPrefix>
+    <VersionSuffix>preview</VersionSuffix>
+  </PropertyGroup>
+
+  <!-- NuGet properties -->
+  <PropertyGroup>
+    <Authors>Apache</Authors>
+    <Description>Apache Arrow is a cross-language development platform for in-memory data. It specifies a standardized language-independent columnar memory format for flat and hierarchical data, organized for efficient analytic operations on modern hardware. It also provides computational libraries and zero-copy streaming messaging and interprocess communication.</Description>
```

Comment:
I'm having trouble parsing the last sentence.  How does Arrow provide interprocess communication?  This package contains "computational libraries"? Etc.

Reply:
Some of the project's libraries have computational libraries, though this one may not yet. The zero-copy read capabilities in the project serve for IPC use cases

Reply:
I stole those three lines from the first paragraph on:

https://arrow.apache.org/

I can drop that last sentence, or just the "computational libraries" piece?

Or I can just use the tag line from https://arrow.apache.org/

> A cross-language development platform for in-memory data

But that doesn't seem like a very good description to me.

Reply:
> The zero-copy read capabilities in the project serve for IPC use cases

Sure.  It's just that the way this is worded, to me I read it as "It also provides … interprocess communication", which suggests that there's some IPC mechanism as part of this library, e.g. named pipes, sockets, shared memory, etc.  This library provides a data serialization format that can be used in the context of such an IPC mechanism to share the data between processes, but it doesn't actually provide the IPC mechanism.  That's what I find confusing.

Reply:
I've removed the last sentence of the description since the C# library doesn't do these things.

@wesm if you have any feedback on this Description, I can definitely make any change you'd like.